.github/workflows/dockerhub.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish Docker image
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+      - name: Log in to Docker Hub
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        with:
+          images: my-docker-hub-namespace/my-docker-hub-repository
+      - name: Build and push Docker image
+        uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          build-args: |
+              "AWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_ID }}"
+              "AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY }}"
```

Comment:
It seems strange to push these secrets to a public image in the Docker registry, though I'm not knowledgeable in his area. Wouldn't that expose their values to anyone who pulls down the image?

Reply:
Oh, good catch! You're definitely right, anyone who built a container from this image could access all its env vars.

Reply:
Thanks Cecilia, I thought about that, should probably have opened a discussion about it. I dropped a comment related to this on slack here : https://permanent-org.slack.com/archives/C01ARGVA849/p1669070086706879?thread_ts=1669064165.264129&cid=C01ARGVA849

This image would be for internal use and would/should be non public because the `env` vars are inspect-able.

What would should follow probably is to add a job to create a public image, without these secrets. Nonetheless, this particular image most likely must have it's credentials embedded for eternal use.

_NB:_ Even though it's possible to pass `env` vars to containers in this case, the image building involves writing the secrets to files especially for `~/.aws`